sample-service/src/main/java/uk/ac/ox/ndph/mts/sample_service/client/WebClientConfig.java

```diff
@@ -0,0 +1,60 @@
+package uk.ac.ox.ndph.mts.sample_service.client;
+
+import io.netty.channel.ChannelOption;
+import io.netty.handler.timeout.ReadTimeoutHandler;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.client.reactive.ClientHttpConnector;
+import org.springframework.http.client.reactive.ReactorClientHttpConnector;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.netty.http.client.HttpClient;
+
+import java.util.concurrent.TimeUnit;
+
+@Configuration
+@ConfigurationProperties(prefix = "http")
+public class WebClientConfig {
```

Comment:
Do we need this WebClientConfig?  Could we just use webclient defaults?

Reply:
I just aligned with how it is done in practitioner service, eventually I understood all services clients are going to be extracted to an external lib, so I wanted it to use it the same way and also for it to be with the same structure

Reply:
Role-service and init-service might be better models for making rest requests?  I haven't looked at practitioner-service - is this class redundant there too or does it do something which overrides defaults?

Reply:
I would leave it as-is for now, since we are going to replace this eventually with the shared client code in fhir-client.

Reply:
ok